Documentation/hotdoc/gen-hotdoc.sh

```diff
@@ -23,4 +32,7 @@ echo "Dependencies file path: $deps_file_path"
 hotdoc run -i index.md -o Documentation/NNStreamer-doc --sitemap=Documentation/hotdoc/sitemap.txt --deps-file-dest=$deps_file_path \
            --html-extra-theme=Documentation/hotdoc/theme/extra --project-name=NNStreamer --project-version=$v
 
+rm *.md
```

Comment:
IMO, checking for the return value of the above ```hotdoc``` command  and the generated files' integrity need to be added here.

Reply:
Thanks for the suggestion. Add log message and file.